opendatakit/briefcase
# Issue 479 reload cache button

Closes #479

This PR adds a new button to the Settings tab that lets the user reload the form cache. This is useful when the user needs to manually add or remove any form in Briefcase's current storage directory.

![image](https://user-images.githubusercontent.com/205913/42159781-447e298e-7df5-11e8-800f-00382816dfd1.png)

#### What has been done to verify that this works as intended?
- Set a storage directory. 
- Check that a form is listed in the push and export tabs.
- Remove the form folder manually
- Click on the reload cache button (verify that a dialog confirms the operation)
- Check that the form is no longer present in the push and export tabs.

#### Why is this the best possible solution? Were any other approaches considered?
This PR introduces UI changes only. All the underlying mechanisms already existed. It's a fairly straightforward change.

#### Are there any risks to merging this code? If so, what are they?
Nope.

#### Does this change require updates to documentation? If so, please file an issue at https://github.com/opendatakit/docs/issues/new and include the link below.
Nope.
